### PR TITLE
12.0 c6e7e846da21776622829dd972eba34049060fd6

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_invoice_pay/doc/changelog.rst [hunk 1]
```diff
@@ -21,6 +21,7 @@
 `1.2.1`
 -------
 
+**Fix:** The change value is not saved on the invoice after payment
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_invoice_pay/doc/changelog.rst:24]: :+1: checked

File: pos_journal_pin/doc/changelog.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+`1.0.0`
+-------
+
+- **Init version**
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_journal_pin/doc/changelog.rst:4]: :o:

File: pos_menu/doc/changelog.rst [hunk 3]
```diff
@@ -1,4 +1,4 @@
 `1.0.0`
 -------
 
-- Init version
+- **Init version**
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_menu/doc/changelog.rst:4]: :o:

File: pos_product_available/doc/changelog.rst [hunk 4]
```diff
@@ -1,7 +1,7 @@
-.. _changelog:
+`1.1.0`
+-------
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_product_available/doc/changelog.rst:2]: :o:

File: pos_longpolling/doc/changelog.rst [hunk 5]
```diff
@@ -1,3 +1,8 @@
+`2.2.0`
+-------
+
+- **IMP:** *Autostart longpolling* field is hidden on the `POS config` form to prevent possible misconfigurations.
+
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_longpolling/doc/changelog.rst:10]: The option is not available in debug mode.

File: pos_product_available_negative/doc/changelog.rst [hunk 6]
```diff
@@ -1,7 +1,14 @@
-.. _changelog:
+`1.1.1`
+-------
+
+- **Fix:** Cashier was changed after giving the authorization of the sale
+
+`1.1.0`
+-------
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_product_available_negative/doc/changelog.rst:7]: :+1: checked

File: pos_multi_session_sync/doc/changelog.rst [hunk 7]
```diff
@@ -1,3 +1,8 @@
+`1.0.7`
+-------
+- **Fix:** Updated 'update_revision_ID' message
+- **Fix:** Possible error with multiple multi sessions with the same multi session ID
+
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_multi_session_sync/doc/changelog.rst:5]: :+1: checked

File: pos_multi_session_restaurant/doc/changelog.rst [hunk 8]
```diff
@@ -1,5 +1,11 @@
+`3.3.2`
+-------
+
+**Fix:** Error on starting a multisession without any floors
+
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_multi_session_restaurant/doc/changelog.rst:5]: :-1: checked

File: pos_longpolling/doc/changelog.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (KolushovAlexandr) [reply pos_longpolling/doc/changelog.rst:10]: it's hidden in pos_longpolling. The option becomes visible after installing pos_multi_session

File: pos_multi_session_restaurant/doc/changelog.rst (hunk 8 above)
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_multi_session_restaurant/doc/changelog.rst:5]: OK. The POS must be closed before changing multisession configuration

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Comment by em230418** [comment]
Куда-то пропал модуль `pos_disable_payment`. Он является зависимостью `pos_disable_payment_restaurant` и его демки как-то использует `pos_multi_session`.

**Comment by KolushovAlexandr** [comment]
@em230418 The module is not maintained since Odoo 12.0. All changes of the module have been migrated to  `Point of Sale - Extra Access Right <https://github.com/OCA/pos/tree/12.0/pos_access_right/>`__ module. Read more `here. <https://github.com/OCA/pos/issues/232>`__
https://github.com/it-projects-llc/pos-addons/blame/11.0/pos_disable_payment/README.rst#L39

pos_disable_payment_restaurant is not installable https://github.com/it-projects-llc/pos-addons/blob/12.0/pos_disable_payment_restaurant/__manifest__.py

pos_multi_session does not depend on pos_disable_payment

**Comment by em230418** [comment]
> pos_disable_payment_restaurant is not installable https://github.com/it-projects-llc/pos-addons/blob/12.0/pos_disable_payment_restaurant/__manifest__.py

Понял. Не посмотрел.

> pos_multi_session does not depend on pos_disable_payment

Я это имел ввиду: https://github.com/it-projects-llc/pos-addons/blob/12.0/pos_multi_session/__manifest__.py#L48

**Review by em230418 — APPROVED** [review]

**Comment by em230418** [comment]
I approve to merge it now

**Pull request merged**